### DISCUSSION TITLE
NSPanel and DS18B20

### DISCUSSION THREAD

**Discussion by gonzzzales** [arendst/Tasmota#15638] [Q&A] [answered]
Is it possible to use DS18B20 with Sonoff NSPanel. I tryed to connect it to GPIO23, but i dont see any messages in console. I need to configure it?

**Comment by barbudor** [comment, marked as answer]
Indeed connecting doesn't make it appear magically unless you co figure gpio23 to 'DS18x20'

**Reply by sfromis** [reply]
You likely also need a pullup resistor.

**Comment by gonzzzales** [comment]
Yes, fogot pullup resistor. Thanks a lot 😁
